torch/csrc/jit/backends/backend.h

```diff
@@ -28,6 +28,10 @@ class backend {
     static auto cls =
         torch::class_<TBackendInterface>(detail::kBackendsNamespace, name)
             .def(torch::init<>())
+            ._def_unboxed(
+                "is_available",
+                detail::getIsAvailableFunc<TBackendInterface>(),
+                detail::getIsAvailableSchema())
```

Comment:
I'm not sure if you meant to say "this is BC", but that's right, this is BC because serialized methods have logic serialized that just depends on two methods in the bound object: compile and execute, which didn't change.